python/mxnet/initializer.py

```diff
@@ -464,6 +464,11 @@ def __init__(self, value):
     def _init_weight(self, _, arr):
         arr[:] = self.value
 
+    def dumps(self):
+        if not np.isscalar(self._kwargs['value']):
+            self._kwargs['value'] = self._kwargs['value'].asnumpy().tolist()
```

Comment:
if the argument passed to the `constant initializer` is of type `numpy.ndarray` then the `asnumpy` call will fail here. for instance a usage such as this - 
```python
sym_var = mx.sym.var('test', init=mx.init.Constant(np.ones((2,3))))
```

Reply:
Fixed!